openood/networks/resnet18_32x32.py

```diff
@@ -107,7 +117,7 @@ def _make_layer(self, block, planes, num_blocks, stride):
             self.in_planes = planes * block.expansion
         return nn.Sequential(*layers)
 
-    def forward(self, x, return_feature=False, return_feature_list=False):
+    def forward(self, x, return_feature_list=False):
```

Comment:
why delete `return_feature` option since other methods might require

Reply:
I mimic wrn to put feature in feature_list. If it is not needed, it can remain the original form since VOS doesn't use resnet 32